### PR TITLE
can delete comments

### PR DESCRIPTION
**Pull request by Matt-Tan15**

### CODE REVIEW

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks good

File: src/controllers/comment.controller.ts [hunk 1]
```diff
@@ -4,16 +4,27 @@ import { BaseCommentDto } from '@/dtos';
...
+      res.status(200).json({ data, message: 'success' });
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  public deleteComment = async (req: RequestWithUser, res: Response, next: NextFunction) => {
+    try {
+      const commentId = Number(req.params.comment_id);
+      const userId = Number(req.user.id);
```
Comment (DLiSWE) [inline src/controllers/comment.controller.ts:21]: userId is already a number.

**Review by ZohaibManzoor00 — APPROVED** [review]
looks good

**Review by DLiSWE — APPROVED** [review]
LGTM!

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks good

**Pull request merged**